doc/classes/AnimationNodeStateMachineTransition.xml

```diff
@@ -10,7 +10,11 @@
 	</methods>
 	<members>
 		<member name="advance_condition" type="String" setter="set_advance_condition" getter="get_advance_condition">
-			Turn on auto advance when this condition is set. This is a custom text field that can be filled with a variable name. The variable can be modified from code.
+            Turn on auto advance when this condition is set. The provided name will become a boolean parameter on the [AnimationTree] that can be controlled from code (<link>https://docs.godotengine.org/en/latest/tutorials/animation/animation_tree.html#controlling-from-code</link>). For example, if [member AnimationTree.tree_root] is an [AnimationNodeStateMachine] and [member advance_condition] is set to "idle":
+
```

Comment:
A line break already creates a new paragraph. An additional empty line just produces an additional empty paragraph, so its unneeded :)

Reply:
Is this the correct way to create a link? Shouldn't it be [url]....[/url]?